.github/workflows/sdk-update-api-spec.yaml

```
@@ -0,0 +1,38 @@
+name: SDK Update API Spec
+
+on:
+  schedule:
+    - cron: '0 12 * * 3'
+  workflow_dispatch: {}
+
+jobs:
+  api-spec-update:
+    runs-on: ubuntu-20.04
+    if: ${{ github.repository == 'jellyfin/jellyfin-sdk-kotlin' }}
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Setup Gradle cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-java-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-java-gradle-
+            ${{ runner.os }}-java-
+      - name: Update generated Jellyfin API spec
+        uses: technote-space/create-pr-action@v2
+        with:
+          EXECUTE_COMMANDS: ./gradlew --build-cache --no-daemon --info updateApiSpecStable
+          COMMIT_MESSAGE: 'update generated Jellyfin API spec'
+          COMMIT_NAME: 'jellyfin-bot'
+          COMMIT_EMAIL: 'team@jellyfin.org'
+          PR_BRANCH_PREFIX: 'chore/'
+          PR_BRANCH_NAME: 'api-gen-update-${PR_ID}'
```

```suggestion
          PR_BRANCH_PREFIX: 'openapi-update-'
          PR_BRANCH_NAME: '${PR_ID}'
```

The gradlew updates look like `gradlew-update-7.0`